ports/operationsteel/Operation S.T.E.E.L.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+if [ -f "$controlfolder/device_info.txt" ]; then
+    source "$controlfolder/device_info.txt"
+else
+    echo "Error: device_info.txt not found in $controlfolder"
+    exit 1
+fi
+
+if [ -f "$controlfolder/mod_${CFW_NAME}.txt" ]; then
+    source "$controlfolder/mod_${CFW_NAME}.txt"
+else
+    echo "Warning: mod_${CFW_NAME}.txt not found in $controlfolder"
+fi
+
+get_controls
+
+GAMEDIR=/$directory/ports/operationsteel/
+CONFDIR="$GAMEDIR/conf/"
+
+# Ensure the conf directory exists
+mkdir -p "$GAMEDIR/conf"
+
+# Set the XDG environment variables for config & savefiles
+export XDG_CONFIG_HOME="$CONFDIR"
+export XDG_DATA_HOME="$CONFDIR"
+
+cd $GAMEDIR
+
+# Logging setup
+exec > >(tee -a "$GAMEDIR/log.txt") 2>&1
```

Comment:
Wrong logging statement, I suggest you study our packaging guide with the example scripts again. 
We have a section where each function is explained.